ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -492,6 +492,68 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: tempo-operator
+  cron: 0 23 * * 1-5
+  steps:
+    test:
+    - as: tempo-operator
+      commands: |
+        rebasebot --source https://github.com/grafana/tempo-operator:main \
+                  --dest openshift/grafana-tempo-operator:main \
+                  --rebase IshwarKanse/tempo-operator:rebase-bot-main \
+                  --update-go-modules \
+                  --github-app-key /secrets/rebasebot/cloud-team-openshift-bot-key \
+                  --github-cloner-key /secrets/dt-secrets/dt-cloner \
+                  --github-cloner-id 811756 \
+                  --git-username IshwarKanse \
```

Comment:
Jut a suggestion: It doesn't have to be a valid github username.
And since it will be a bot and not yourself to commit these, I'd probably avoid using your username.
You could still call it `tracing-team-rebase-bot` or something along these lines